nextflow.config

```
@@ -102,7 +102,7 @@ params {
     shortread_hostremoval_index            = null
     longread_hostremoval_index             = null
     save_hostremoval_index                 = false
-    save_hostremoval_bam                = false
+    save_hostremoval_bam                  = false
```

```suggestion
    save_hostremoval_bam                     = false
```

I missed some spaces here but now it should be fine :)